### ISSUE TITLE
New .mat file in test script is causing script to fail

### ISSUE THREAD

**Issue by jdesbonnet** [opencobra/COBRA.jl#82]
Ref pull request #80  merged to master branch in around 2020-09-07

The new ecoli model at 
http://bigg.ucsd.edu/static/models/e_coli_core.mat (size 114736) is causing test script to fail.

The older URL 
https://github.com/opencobra/COBRA.models/raw/master/mat/ecoli_core_model.mat does work but is a considerably smaller file of 8818 bytes)

URL is specified in file test/getTestModel.jl

**Comment by wrcannon** [comment]
I have a similar problem after downloading COBRA via Pkg.add("COBRA"):

ERROR: LoadError: HTTP.ExceptionRequest.StatusError(404, "GET", "/LCSB-BioCore/COBRA.models/raw/master/mat/ecoli_core_model.mat", HTTP.Messages.Response:
"""
HTTP/1.1 404 Not Found
date: Wed, 23 Sep 2020 15:30:25 GMT
content-type: text/html; charset=utf-8
server: GitHub.com
status: 404 Not Found
vary: X-PJAX, Accept-Encoding, Accept, X-Requested-With
link: <https://github.githubassets.com/_error.js>; rel=preload; as=script; nopush
cache-control: no-cache
strict-transport-security: max-age=31536000; includeSubdomains; preload
x-frame-options: deny
x-content-type-options: nosniff
x-xss-protection: 1; mode=block
referrer-policy: origin-when-cross-origin, strict-origin-when-cross-origin
expect-ct: max-age=2592000, report-uri="https://api.github.com/_private/browser/errors"
content-security-policy: default-src 'none'; base-uri 'self'; block-all-mixed-content; connect-src 'self' uploads.github.com www.githubstatus.com collector.githubapp.com api.github.com www.google-analytics.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com; frame-ancestors 'none'; frame-src render.githubusercontent.com; img-src 'self' data: github.githubassets.com identicons.github.com collector.githubapp.com github-cloud.s3.amazonaws.com *.githubusercontent.com; manifest-src 'self'; media-src 'none'; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; worker-src github.com/socket-worker.js gist.github.com/socket-worker.js
Content-Length: 176100
X-GitHub-Request-Id: DFDB:91E1:6BFCC:9AA73:5F6B6A33






<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
  <link rel="dns-prefetch" href="https://github.githubassets.com">
  <link rel="dns-prefetch" href="https://avatars0.githubusercontent.com">
  <link rel="dns-prefetch" href="https://avatars1.githubusercontent.com">
  <link rel="dns-prefetch" href="https://avatars2.githubusercontent.com">
  <link rel="dns-prefetch" href="https://avatars3.githubusercontent.com">
  <link rel="dns-prefetch" href="https://github-cloud.s3.amazonaws.com">
  <link rel="dns-prefetch" href="https://user-images.githubusercontent.com/">



  <link crossorigin="anonymous" media="all" integrity="sha512-lXny7HsXlG+Rh3IMsYRkGDZDcGBymPbtznhrj7Yy1Bpa8aSPuw16+crDitBAvaIozcBsaG2dERaJrWKn858hYA==" rel="stylesheet" href="https://github.githubassets.com/assets/frameworks-9579f2ec7b17946f9187720cb1846418.css" />
  <link crossorigin="anonymous" media="all" integrity="sha512-JmyzpcT3FELE4oXIsxT18uanyyF3Y0tyzXSpJhKdly8aiI7i8exXFaCj1sbeCDvxm/D8viQ6ApnNFXR+/
⋮
176100-byte body
""")
Stacktrace:
 [1] request(::Type{ExceptionLayer{ConnectionPoolLayer{StreamLayer{Union{}}}}}, ::HTTP.URIs.URI, ::Vararg{Any,N} where N; kw::Base.Iterators.Pairs{Symbol,Union{Nothing, Bool},Tuple{Symbol,Symbol},NamedTuple{(:iofunction, :reached_redirect_limit),Tuple{Nothing,Bool}}}) at /Users/user1/.julia/packages/HTTP/IAI92/src/ExceptionRequest.jl:22
 [2] (::Base.var"#56#58"{Base.var"#56#57#59"{ExponentialBackOff,HTTP.RetryRequest.var"#2#3"{Bool,HTTP.Messages.Request},typeof(HTTP.request)}})(::Type{T} where T, ::Vararg{Any,N} where N; kwargs::Base.Iterators.Pairs{Symbol,Union{Nothing, Bool},Tuple{Symbol,Symbol},NamedTuple{(:iofunction, :reached_redirect_limit),Tuple{Nothing,Bool}}}) at ./error.jl:288
 [3] #request#1 at /Users/user1/.julia/packages/HTTP/IAI92/src/RetryRequest.jl:44 [inlined]
 [4] request(::Type{MessageLayer{RetryLayer{ExceptionLayer{ConnectionPoolLayer{StreamLayer{Union{}}}}}}}, ::String, ::HTTP.URIs.URI, ::Array{Pair{SubString{String},SubString{String}},1}, ::Array{UInt8,1}; http_version::VersionNumber, target::String, parent::Nothing, iofunction::Nothing, kw::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol},NamedTuple{(:reached_redirect_limit,),Tuple{Bool}}}) at /Users/user1/.julia/packages/HTTP/IAI92/src/MessageRequest.jl:51
 [5] request(::Type{BasicAuthLayer{MessageLayer{RetryLayer{ExceptionLayer{ConnectionPoolLayer{StreamLayer{Union{}}}}}}}}, ::String, ::HTTP.URIs.URI, ::Array{Pair{SubString{String},SubString{String}},1}, ::Array{UInt8,1}; kw::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol},NamedTuple{(:reached_redirect_limit,),Tuple{Bool}}}) at /Users/user1/.julia/packages/HTTP/IAI92/src/BasicAuthRequest.jl:28

**Comment by jdesbonnet** [comment]
Replacing "LCSB-BioCore" with "opencobra" in the URL should cause that test to succeed (the repositories moved from LCSB-BioCore to opencobra on github.com earlier this year.

**Comment by wrcannon** [comment]
Yes, definitely. I just wanted to point out that the test framework needs to be updated.
Bill

**Comment by jdesbonnet** [comment]
@rmtfleming Hi Ronan, what's the best .mat file to use here? The original one when corrected for the github move completes the test sucessfully. But the new one introduced about 2 weeks ago ( http://bigg.ucsd.edu/static/models/e_coli_core.mat ) currently fails the test.

**Comment by rmtfleming** [comment]
https://github.com/opencobra/COBRA.models/raw/master/mat/ecoli_core_model.mat

On Thu, 24 Sep 2020 at 15:38, Joe Desbonnet <notifications@github.com> wrote:
>
> @rmtfleming Hi Ronan, what's the best .mat file to use here? The original one when corrected for the github move completes the test sucessfully. But the new one introduced about 2 weeks ago ( http://bigg.ucsd.edu/static/models/e_coli_core.mat ) currently fails the test.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.



-- 
--
Mr. Ronan MT Fleming B.V.M.S. Dip. Math. Ph.D.
----------------------------------------------------------------------------
Assistant Professor,
Division of Systems Biomedicine and Pharmacology,
Leiden Academic Centre for Drug Research,
Faculty of Science,
Leiden University.
https://www.universiteitleiden.nl/en/staffmembers/ronan-fleming
&
H2020 Project Coordinator,
Systems Medicine of Mitochondrial Parkinson’s Disease,
http://sysmedpd.eu
&
Adjunct Lecturer,
School of Medicine,
National University of Ireland, Galway.
----------------------------------------------------------------------------
Peer-reviewed publications: https://goo.gl/FZPG23
Mobile:  +353 852 109 806
Skype: ronan.fleming
----------------------------------------------------------------------------
(This message is confidential and may contain privileged information.
It is intended for the named recipient only. If you receive it in
error please notify me and permanently delete the original message and
any copies.)
